Mlem/App/Utility/Extensions/Content Models/DeletableProviding+Extensions.swift

```diff
@@ -12,7 +12,7 @@ extension DeletableProviding {
         if feedback.contains(.toast), !deleted {
             let task = toggleDeleted()
             Task {
-                let result = try await task.result.get()
+                let result = await task.result.get()
```

Comment:
This change causes build failure on Xcode 15.4--I'm not entirely sure why it built for me last review, but I can't get it to build now even on a totally clean clone

```suggestion
                let result = try await task.result.get()
```